### PR TITLE
xds: rename resolver scheme to xds

### PR DESCRIPTION
**Pull request by menghanl**
Also keeping "-experimental" for one more release

### CODE REVIEW

File: test/kokoro/xds.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ GRPC_GO_LOG_VERBOSITY_LEVEL=99 GRPC_GO_LOG_SEVERITY_LEVEL=info \
     --gcp_suffix=$(date '+%s') \
     --verbose \
     --client_cmd="grpc-go/interop/xds/client/client \
-      --server=xds-experimental:///{server_uri} \
+      --server=xds:///{server_uri} \
```
Comment (menghanl) [inline test/kokoro/xds.sh:31]: @ericgribkoff Is this the only change for the interop tests to use scheme `xds` instead of `xds-experimental`?

Comment (ericgribkoff) [reply test/kokoro/xds.sh:31]: Yes

**Comment by menghanl** [comment]
We will keep `-experimental` (so we have both) for a release.

File: xds/internal/resolver/xds_resolver_test.go [hunk 2]
```diff
@@ -54,6 +54,15 @@ var (
 	target = resolver.Target{Endpoint: targetStr}
 )
 
+func TestRegister(t *testing.T) {
+	for _, s := range []string{"xds", "xds-experimental"} {
+		b := resolver.Get(s)
+		if b == nil {
```
Comment (dfawley) [inline xds/internal/resolver/xds_resolver_test.go:60]: `if b == nil || (grpc.Version >= "1.30-dev" && s === "xds-experimental")`

OK that's probably not worth doing.

Comment (menghanl) [reply xds/internal/resolver/xds_resolver_test.go:60]: What is it for?

Comment (dfawley) [reply xds/internal/resolver/xds_resolver_test.go:60]: To ensure the "for a release" in:

> We will keep -experimental (so we have both) for a release.

Comment (menghanl) [reply xds/internal/resolver/xds_resolver_test.go:60]: I just set a reminder to myself, to remove it week after the next release

**Pull request merged**